index.ts

```diff
@@ -32,6 +32,7 @@ interface Article {
   slug: string
   labels: Label[]
   highlights: Highlight[]
+  updatedAt: string
```

Comment:
Just some feedback...probably a different type should be used between when you calling loadArticles vs loadArticle because of the difference in the graphql queries.......and not all properties are always there...it's really dependent on which function you are doing.......

Reply:
Nice catch! In the backend of Omnivore, actually we use the same struct of Article for both API but some properties are required and some are optional depending on the query